sdk/core/azure-core-test/src/main/java/com/azure/core/test/policy/RecordNetworkCallPolicy.java

```diff
@@ -49,10 +49,15 @@
     private static final String STATUS_CODE = "StatusCode";
```

Comment:
Missing copy authorization changes in this PR?

Reply:
I was thinking to having the core-test change only in this PR. But I can add the FR changes in this PR as well if you want.

Reply:
I think go ahead and do it. It will help to confirm the changes and the need for this change.